### ISSUE TITLE
Grid not working in Firefox

### ISSUE THREAD

**Issue by kelvinaero** [foundation/foundation-emails#314]
**How can we reproduce this bug?**
We noticed that the grid is not displaying correctly on firefox(See attached)
![email-grid-problem](https://cloud.githubusercontent.com/assets/9321587/14186629/4235634e-f74d-11e5-93a3-6f0d5394684f.jpg)

We understand that Zurb for email is intended for email clients and not for browser, but we have a lot of our users using gmail on firefox and reporting this issue.

In the previous version our emails where 580px wide. We always followed the responsive image rules of making the image 580px placing **width="280"** on images going on **six columns**.

Here is the inky code

```
<row>
    <columns small="12" large="12">
      <img src="header-offertas-spa-sjo.jpg" alt="Compra por categorias*">
    </columns>
  </row>

  <row>
    <columns>
      <a
          href="#"
          target="_blank">
        <img  src="brand-1-spa-sjo-asics.jpg" alt="Asics"></a>
    </columns>
    <columns >
      <a
          href="#"
          target="_blank">
        <img src="brand-2-spa-sjo-bulova.jpg" alt="Bulova"></a>
    </columns>
  </row>

  <row>
    <columns>
      <a
          href="#"
          target="_blank">
        <img src="brand-3-spa-sjo-calvin-klein.jpg" alt="Calvin Klein">
      </a>
    </columns>
    <columns>
      <a href="#" target="_blank">
        <img src="brand-4-spa-sjo-puma.jpg" alt="PUMA">
      </a>
    </columns>
  </row>
```

Any ideas?

**Comment by tdhartwick** [comment]
Firefox is especially finicky to spaces within the html itself, the same problem you get with display inline-block. Is the dist code compressed?

**Comment by kelvinaero** [comment]
Yes, it is compressed. Today we updated to the 2.1 and this issue is persistent. In the ink version we can stabilize it in Firefox by repeating width: 280px; inside the inline styles.  With this version(inky) we don't have that control, is there anyway in use custom css styles?

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
